### ISSUE TITLE
Confused about the versions

### ISSUE THREAD

**Issue by derwaldgeist** [Meteor-Community-Packages/meteor-roles#289]
The Github master tells me v3 is the most recent version, yet it’s core code dates back 2 years, whereas v2 seems to be only 10 days old. Why is this so?

**Comment by mitar** [comment]
`master` is staging place for v3 version which is incoming. This is waiting for @SimonSimCity to do it.

**Comment by SimonSimCity** [comment]
Version 1 is what was published on Atmosphere until recently. A version 2 was in the making for quite a while and has been heavily used by people in the community before it was published - https://github.com/Meteor-Community-Packages/meteor-roles/issues/165. In our company the need arose to change the roles package, which is discussed in #270. Out of it came a bunch of PRs which were open and now are awaiting my re-opening. The branching structure changed since we've moved to this repo and I have to reserve some time to merge them in without discarding the changes made in the meanwhile.

Since all previous versions (V1 and V2) still are heavily used and V3 has a different database structure (likewise has V1 compared to V2) we chose to keep all versions active in regards to fixing critical bugs, but V3 and it's successors will be the only versions which will get new features.

More on this in https://github.com/Meteor-Community-Packages/organization/issues/25 and https://github.com/Meteor-Community-Packages/meteor-roles/issues/283

I'll add something into the readme of this repo regarding this.
